src/cdomains/floatDomain.ml

```diff
@@ -125,3 +163,123 @@ module FloatDomTuple = struct
 
   let ne = eval_int_binop ne
 end
+
+module FloatDomImpl = struct
+  include Printable.Std (* for default invariant, tag, ... *)
+  module F1 = FloatInterval
+  open Batteries
+
+  type t = F1.t option [@@deriving to_yojson, eq, ord]
+
+  let name () = "FloatInterval"
```

Comment:
Maybe I don't understand it fully, but should this name not be for the FloatInterval module above?